test/ShallowWrapper-spec.jsx

```diff
@@ -565,6 +565,36 @@ describe('shallow', () => {
       expect(wrapper.find('a[value=true]')).to.have.length(0);
     });
 
+    it('should support attributes with numbers', () => {
+      const wrapper = shallow(
+        <div>
+          <span value-1={1} />
+          <a value-1={false} />
+        </div>,
+      );
+
+      expect(wrapper.find('span[value-1=1]')).to.have.length(1);
```

Comment:
Would that be a breaking change? This existing test would fail.
` it('should support boolean and numeric values for matching props', () => {
      const wrapper = shallow(
        <div>
          <span value={1} />
          <a value={false} />
        </div>,
      );

      expect(wrapper.find('span[value=1]')).to.have.length(1);
`

currently expect(wrapper.find('span[value="1"]')).to.have.length(1); would fail